<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeLoading</class>
 <widget class="QWidget" name="HomeLoading">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Com_Loading_lbAmi">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Com_Loading_lbMessage">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Com_Loading_lbBack">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>Com_Loading_lbBack</zorder>
  <zorder>Com_Loading_lbAmi</zorder>
  <zorder>Com_Loading_lbMessage</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
